trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/deflect_control_surface.py

```diff
@@ -0,0 +1,619 @@
+        y_prime_bc     = VD.YBC[condition]
+        y_prime_b2     = VD.YB2[condition]
+        zeta_prime_b1  = VD.ZB1[condition]
+        zeta_prime_bh  = VD.ZBH[condition]
+        zeta_prime_bc  = VD.ZBC[condition]
+        zeta_prime_b2  = VD.ZB2[condition]
+        xi_prime_ch    = VD.XCH[condition]
+        xi_prime       = VD.XC [condition]
+        y_prime_ch     = VD.YCH[condition]
+        y_prime        = VD.YC [condition]
+        zeta_prime_ch  = VD.ZCH[condition]
+        zeta_prime     = VD.ZC [condition]
+        
+        X_as = np.zeros_like(VD.X[condition_full][:-(n_cw+1)])
+        Y_as = np.zeros_like(VD.Y[condition_full][:-(n_cw+1)])
+        Z_as = np.zeros_like(VD.Z[condition_full][:-(n_cw+1)])
+        
+        
+        for idx_y in range(n_sw):
+            start     , stop      = idx_y*n_cw    , (idx_y+1)*n_cw
+            start_full, stop_full = idx_y*(n_cw+1), (idx_y+1)*(n_cw+1)
+            
+            # pack strip values
+            raw_VD = Data()
+            raw_VD.xi_prime_a1   = xi_prime_a1  [start:stop]
+            raw_VD.xi_prime_ac   = xi_prime_ac  [start:stop]
+            raw_VD.xi_prime_ah   = xi_prime_ah  [start:stop]
+            raw_VD.xi_prime_a2   = xi_prime_a2  [start:stop]
+            raw_VD.y_prime_a1    = y_prime_a1   [start:stop]
+            raw_VD.y_prime_ah    = y_prime_ah   [start:stop]
+            raw_VD.y_prime_ac    = y_prime_ac   [start:stop]
+            raw_VD.y_prime_a2    = y_prime_a2   [start:stop]
+            raw_VD.zeta_prime_a1 = zeta_prime_a1[start:stop]
+            raw_VD.zeta_prime_ah = zeta_prime_ah[start:stop]
+            raw_VD.zeta_prime_ac = zeta_prime_ac[start:stop]
+            raw_VD.zeta_prime_a2 = zeta_prime_a2[start:stop]
+            raw_VD.xi_prime_b1   = xi_prime_b1  [start:stop]
+            raw_VD.xi_prime_bh   = xi_prime_bh  [start:stop]
+            raw_VD.xi_prime_bc   = xi_prime_bc  [start:stop]
+            raw_VD.xi_prime_b2   = xi_prime_b2  [start:stop]
+            raw_VD.y_prime_b1    = y_prime_b1   [start:stop]
+            raw_VD.y_prime_bh    = y_prime_bh   [start:stop]
+            raw_VD.y_prime_bc    = y_prime_bc   [start:stop]
+            raw_VD.y_prime_b2    = y_prime_b2   [start:stop]
+            raw_VD.zeta_prime_b1 = zeta_prime_b1[start:stop]
+            raw_VD.zeta_prime_bh = zeta_prime_bh[start:stop]
+            raw_VD.zeta_prime_bc = zeta_prime_bc[start:stop]
+            raw_VD.zeta_prime_b2 = zeta_prime_b2[start:stop]
+            raw_VD.xi_prime_ch   = xi_prime_ch  [start:stop]
+            raw_VD.xi_prime      = xi_prime     [start:stop]
+            raw_VD.y_prime_ch    = y_prime_ch   [start:stop]
+            raw_VD.y_prime       = y_prime      [start:stop]
+            raw_VD.zeta_prime_ch = zeta_prime_ch[start:stop]
+            raw_VD.zeta_prime    = zeta_prime   [start:stop]
+            
+            # deflect the surface
+            raw_VD = deflect_control_surface_strip(wing, raw_VD, idx_y==0, sym_sign)
```

Comment:
Is idx_y always zero? If so can I change this?

I can't remember what we did here

Reply:
idx_y is only 0 for the first strip of a control surface VLM_wing. Because they tend to be small, they often only have 1 strip, but this isn't guaranteed.

Reply:
So remove this?

Reply:
I mean remove the assumption it's always 0.

If you want it to always check then I'll remove the if statement below.